Library/Homebrew/cmd/update-bash-report.rb

```diff
@@ -0,0 +1,351 @@
+require "cmd/tap"
+require "formula_versions"
+require "migrator"
+require "formulary"
+require "descriptions"
+
+module Homebrew
+  def update_bash_report
```

Comment:
For the time being, let's prevent users calling these command directly. I.e. Adding homebrew developer check.